internal/pkg/cli/task_run.go

```diff
@@ -437,22 +437,21 @@ func BuildTaskRunCmd() *cobra.Command {
 	}
 	cmd := &cobra.Command{
 		Use:   "run",
-		Short: "Run a one-off task",
-		Long:  `Run a one-off task with configurations such as cpu-units, memory, image, etc.`,
+		Short: "Run a one-off task on Amazon ECS.",
 		Example: `
-Run a task with default setting. You will be prompted to specify a task group name and an environment for the tasks to run in.
+Run a task using your local Dockerfile and minimal container sizes. 
```

Comment:
Maybe remove the "and minimal container sizes" bit - it's a little confusing.

Reply:
Done!